src/sdl3_syms.h

```diff
@@ -529,7 +529,7 @@ SDL3_SYM(int,GetDisplayUsableBounds,(SDL_DisplayID a, SDL_Rect *b),(a,b),return)
 SDL3_SYM_PASSTHROUGH(int,GetWindowBordersSize,(SDL_Window *a, int *b, int *c, int *d, int *e),(a,b,c,d,e),return)
 SDL3_SYM_PASSTHROUGH(int,SetWindowOpacity,(SDL_Window *a, float b),(a,b),return)
 SDL3_SYM_PASSTHROUGH(int,GetWindowOpacity,(SDL_Window *a, float *b),(a,b),return)
-SDL3_SYM_PASSTHROUGH(int,SetWindowInputFocus,(SDL_Window *a),(a),return)
+SDL3_SYM(int,SetWindowInputFocus,(SDL_Window *a),(a),return)
```

Comment:
You should just delete this line. The function is no longer in SDL3.

Reply:
Fixed.